controllers/department.js

```diff
@@ -1,31 +1,61 @@
-const Department = require("../models/department");
+const DepartmentService = require("../services/department");
+const EmployeeService = require("../services/employee");
 
-class departmentController {
+class DepartmentController {
+  async createNewDepartment(request, response) {
+    try {
+      const department = await DepartmentService.createNewDepartment(request.body, request.files.picture);
```

Comment:
I do not see any validation. what will be here if post will not contain needed data?